vmm/src/coredump.rs

```diff
@@ -0,0 +1,366 @@
+// Copyright 2022 ZTE Corporation.
+// Portions Copyright 2020 The Chromium OS Authors. All rights reserved.
```

Comment:
Did you import this code from somewhere else? If you wrote it from scratch it should be Apache 2.0 and just the one copyright holder.

Reply:
I wrote this from scratch. I will rewrite this part.